podcasts/DownloadManager.swift

```diff
@@ -459,4 +470,21 @@ class DownloadManager: NSObject, FilePathProtocol {
         downloadTask?.taskDescription = taskId
         downloadTask?.resume()
     }
+
+    func resumeAllBackgrounded() {
+        let queuedEpisodes = DataManager.sharedManager.findEpisodesWhere(customWhere: "episodeStatus == ?", arguments: [DownloadStatus.queued.rawValue])
```

Comment:
All episodes still queued should be retried since they may have been cancelled previously. `addToQueue` will check if there's already a task associated with this episode.